dojson/utils.py

```diff
@@ -44,6 +43,20 @@ def wrapper(self, key, value, **kwargs):
     return wrapper
 
 
+def ignore_list_element(f):
```

Comment:
I'm still not convinced that we need this decorator. Please remove it from this PR and we can start separate discussion on its usage. Thanks